dask/dataframe/core.py

```diff
@@ -2227,6 +2248,19 @@ def meth(self, other, axis='columns', level=None, fill_value=None):
         meth.__doc__ = op.__doc__
         bind_method(cls, name, meth)
 
+    @classmethod
+    def _bind_comparison_method(cls, name, comparison):
+        """ bind comparison method like DataFrame.add to this class """
+
+        def meth(self, other, axis='columns', level=None):
+            if level is not None:
+                raise NotImplementedError('level must be None')
+            axis = self._validate_axis(axis)
+            return elemwise(comparison, self, other, axis=axis)
+
+        meth.__doc__ = comparison.__doc__
+        bind_method(cls, name, meth)
```

Comment:
I modeled this off of a similar _bind_operator_method class method which is implemented separately for Series and DataFrame. I agree moving it to the _Frame object would be less repetitive but if dask moves to 19.0 the Series and DataFrame versions will be slightly different because of the fill value keyword. I'm happy going either way on this though.